blockly-core/core/block_space_editor.js

```diff
@@ -266,13 +288,6 @@ Blockly.BlockSpaceEditor.prototype.flyoutBumpOrDeleteOutOfBoundsBlocks_ = functi
   for (var b = 0, block; block = blocks[b]; b++) {
     var blockXY = block.getRelativeToSurfaceXY();
     var blockHW = block.getHeightWidth();
-    // Have flyout handle any blocks that have been dropped on it
-    if (block.isDeletable() && (Blockly.RTL ?
-        blockXY.x - 2 * metrics.viewLeft - metrics.viewWidth :
-        -blockXY.x) > MARGIN * 2) {
-      this.flyout_.onBlockDropped(block);
-      return;
-    }
```

Comment:
This delete is a combined side-effect of removing the concreteBlocks setting and unifying block deletion logic; the flyout no longer has a need to handle blocks dropped onto it (and if it ever does again, that message should probably be hooked up elsewhere).